Frontend/src/components/AdminDressList.js

```diff
@@ -1,11 +1,21 @@
 import AdminSidebar from "./AdminSidebar";
 import "./AdminDressList.css"
+import { useEffect, useState } from "react";
 function AdminDressList() {
-    const dresses = [
-        { serialNo: 1, id: 201, name: 'Summer Dress', stock: 5 },
-        { serialNo: 2, id: 202, name: 'Evening Gown', stock: 0 },
-        { serialNo: 3, id: 203, name: 'Casual Dress', stock: 10 },
-       ];
+
+    const [allDresses, setAllDresses] = useState("")
+
+    let getAllDresses = async() => {
+        let res = await fetch("/api/v1/dress/")   //backend dress link here
```

Comment:
link -> "/api/v1/suit/"

Also if possible, change the name 'dress' to 'suit' everywhere